src/common/header/shared.h

```diff
@@ -59,12 +59,23 @@ typedef unsigned char byte;
 	#define YQ2_ALIGNAS_TYPE(TYPE)  _Alignas(TYPE)
 	// must be used as prefix (YQ2_ATTR_NORETURN void bla();)!
 	#define YQ2_ATTR_NORETURN       _Noreturn
+#if defined(__GNUC__)
```

Comment:
and while I'm nitpicking, I'd indent this `#if` (and the corresponding `#elif`, `#else` and `#endif`) a by two spaces, like I did below in the `_MSC_VER` -> `_WIN64` case, so it's easier to tell which `#else` (etc) belongs to which `#if`